src/modules/Admin/components/SystemSettings/container.jsx

```diff
@@ -81,6 +81,10 @@ class SystemSettingsBuilder extends ContainerBuilder {
       },
       applySettings: () => {
         dispatchProps.applySettings(() => dispatchProps.loadSystemSettings());
+      },
+      checkAtlas: () => {
+        dispatchProps.checkAtlas()
+          .catch((error) => {alert(error.errors._error)})
```

Comment:
I'd also added condition to mapStateToProps to show loader at the time of request